Specs/Scene/OctahedralProjectedCubeMapSpec.js

```diff
@@ -14,7 +14,7 @@ describe(
     var octahedralMap;
 
     var environmentMapUrl =
-      "./Data/EnvironmentMap/kiara_6_afternoon_2k_ibl.ktx";
+      "./Data/EnvironmentMap/kiara_6_afternoon_2k_ibl.ktx2";
```

Comment:
This seems to break the first two specs because the numbers don't match anymore. Will have to look into this; the aspect ratio of the new texture width and height in `"creates a packed texture with the right dimensions"` matches the old ratio, but the resolution is higher.

Reply:
update: as of 11edcf9 the texture dimensions test passes but I had to bump the RGB epsilon from 5 to 6 in the correct projection spec.